agents/agents/executor/executor.go

```diff
@@ -832,11 +867,14 @@ func (e Executor) receiveLogs(ctx context.Context, chainID uint32) error {
 //
 //nolint:gocognit,cyclop
 func (e Executor) executeExecutable(parentCtx context.Context, chainID uint32) (err error) {
```

Comment:
[nit] no  need for parentCtx if we're not wrapping

Reply:
wym? we do wrap, just later on in the function

Reply:
nah, it remains parentCtx and never gets wrapped into ctx